[{"name": "test.test_interleave.Interleave.test_calls", "status": "failed", "message": "Expected 'zip' to have been called once. Called 0 times.", "passed": false, "points": ["p01-11.1"], "backtrace": ["  File \"/usr/lib/python3.12/unittest/case.py\", line 58, in testPartExecutor\n    yield\n", "  File \"/usr/lib/python3.12/unittest/case.py\", line 634, in run\n    self._callTestMethod(testMethod)\n", "  File \"/usr/lib/python3.12/unittest/case.py\", line 589, in _callTestMethod\n    if method() is not None:\n       ^^^^^^^^\n", "  File \"/home/marco/python-data-analysis-helsinki/01-python/mooc-data-analysis-with-python-2024-2025/part01-e11_interleave/test/test_interleave.py\", line 46, in test_calls\n    pzip.assert_called_once()\n", "  File \"/usr/lib/python3.12/unittest/mock.py\", line 923, in assert_called_once\n    raise AssertionError(msg)\n"]}, {"name": "test.test_interleave.Interleave.test_first", "status": "failed", "message": "Lists differ: [] != [1, 20, 'a', 2, 30, 'b', 3, 40, 'c']\n\nSecond list contains 9 additional elements.\nFirst extra element 0:\n1\n\n- []\n+ [1, 20, 'a', 2, 30, 'b', 3, 40, 'c'] : Incorrect result for input lists [1, 2, 3], [20, 30, 40], ['a', 'b', 'c']!", "passed": false, "points": ["p01-11.1"], "backtrace": ["  File \"/usr/lib/python3.12/unittest/case.py\", line 58, in testPartExecutor\n    yield\n", "  File \"/usr/lib/python3.12/unittest/case.py\", line 634, in run\n    self._callTestMethod(testMethod)\n", "  File \"/usr/lib/python3.12/unittest/case.py\", line 589, in _callTestMethod\n    if method() is not None:\n       ^^^^^^^^\n", "  File \"/home/marco/python-data-analysis-helsinki/01-python/mooc-data-analysis-with-python-2024-2025/part01-e11_interleave/test/test_interleave.py\", line 26, in test_first\n    self.assertEqual(result, [1, 20, 'a', 2, 30, 'b', 3, 40, 'c'], msg=\"Incorrect result for input lists %s, %s, %s!\" % (L1, L2, L3))\n"]}, {"name": "test.test_interleave.Interleave.test_random", "status": "failed", "message": "0 != 200 : Incorrect result list length!", "passed": false, "points": ["p01-11.1"], "backtrace": ["  File \"/usr/lib/python3.12/unittest/case.py\", line 58, in testPartExecutor\n    yield\n", "  File \"/usr/lib/python3.12/unittest/case.py\", line 634, in run\n    self._callTestMethod(testMethod)\n", "  File \"/usr/lib/python3.12/unittest/case.py\", line 589, in _callTestMethod\n    if method() is not None:\n       ^^^^^^^^\n", "  File \"/home/marco/python-data-analysis-helsinki/01-python/mooc-data-analysis-with-python-2024-2025/part01-e11_interleave/test/test_interleave.py\", line 36, in test_random\n    self.assertEqual(len(result), n * size, msg=\"Incorrect result list length!\")\n"]}]